### DISCUSSION TITLE
Retrieving Content Values From Table

### DISCUSSION THREAD

**Discussion by tardisconsole** [DomCR/ACadSharp#312] [Q&A]
Hi,

I'm writing an import tool that simply needs to parse some content text contained in a table inside a DWG.  
Using AcadSharp, I can see virtually everything except the table data.  Reading some other discussions relating to ACAD_TABLE, it seems this has not yet been implemented.  

Doing a foreach through the document entities, I have found BlockRecord entries that match the majority of the content text contained in the table, but there are cases where the content text in the table is an empty string.  These entries are not included in the BlockRecord list when reading in the DWG file.  Because of this, it's impossible to tell where the empty strings are when running through the list.

Will ACAD_TABLE support be available soon? Just a list of entities with grid row/column references and the content text would essentially solve my issue.
Or alternatively, is there a way of including the empty BlockRecord entries (if they exist)?

Thank you.

**Comment by DomCR** [comment]
Hi @tardisconsole,

Unfortunately the ACAD_TABLE is not implemented yet, I'll open an issue linked to this discussion to start the work.

About an alternative way of retrieving the Text, I'm not sure how to do it right now, I haven't work with the tables and I'm not sure how the data is organized, I'll go back to you if I find something.

**Reply by tardisconsole** [reply]
Hi @DomCR,

Thank you for opening the issue, being able to read the table data in some form would be extremely helpful.
